src/main/java/langley/storage/Storage.java

```diff
@@ -0,0 +1,62 @@
+package langley.storage;
+
+import langley.tasks.Task;
+
+import java.io.BufferedWriter;
+import java.io.File;
+import java.io.FileWriter;
+import java.io.IOException;
+import java.io.BufferedReader;
+import java.io.FileReader;
+import java.util.ArrayList;
+
+public class Storage {
```

Comment:
I noticed the missing Javadoc in several other places too. Consider adding them to make your code even better